app/assets/javascripts/short-form/ShortFormApplicationController.js.coffee

```diff
@@ -75,6 +75,12 @@ ShortFormApplicationController = (
 
   $scope.emailRegex = SharedService.emailRegex
 
+  $scope.propertyCardImageURL = ->
+    if _.isArray($scope.listing.Listing_Images)
+      return $scope.listing.Listing_Images[0].Image_URL
```

Comment:
The [directory](https://github.com/SFDigitalServices/sf-dahlia-web/blob/main/app/javascript/modules/listings/SharedHelpers.tsx#L70) and [details](https://github.com/SFDigitalServices/sf-dahlia-web/blob/main/app/javascript/modules/listingDetails/ListingDetailsImageCard.tsx#L19) page are both using the field `displayImageURL`. Looking at some responses it looks to be the same URL as `Image_URL` but to ensure we are using the same images I think we can use the same field.